.travis.yml

```diff
@@ -14,11 +16,16 @@ before_install:
   - cp config/secrets.travis.yml config/secrets.yml
   - git submodule init
   - git submodule update --depth=10 -- red5
+  - rvm use 2.3.3@global
```

Comment:
This appears to be required to revert the bundler version in the next line. Here's what happens if you don't have it: https://travis-ci.org/ualbertalib/avalon/jobs/264540700